packages/backend/src/controllers/authentication.ts

```diff
@@ -467,7 +499,7 @@ const genericOidcHandler =
 
             const openIdUser = createOpenIdUserFromProfile(
                 profile,
-                issuer,
+                issuerOverride || issuer,
```

Comment:
`openid-client` returns a `TokenSet` as the issuer (instead of a string), which breaks some of the checks downstream (e.g `UserService` checks if something is an openId user, including by checking if issuer is a string.

Reply:
Do we need to update the type in `packages/backend/src/@types/passport-openidconnect.d.ts`?

Reply:
Didn't realize we managed our own type implementation -- I'll move these changes there!